packages/grafana-ui/src/components/Forms/Checkbox.mdx

```diff
@@ -5,6 +5,12 @@ import { Checkbox } from './Checkbox';
 
 # Checkbox
 
+### When to use
+
+`Checkbox` can be used in groups, where the single checkboxes have no dependencies. That means that selecting one doesn’t affect any other `Checkbox`. Checked represents true, un-checked represent false. So you can use them to select a binary option or multiple options in a set. When adding a description to your `Checkbox`, use positive commands, not negatives like "do not …".
```

Comment:
`can` or `should` be used in group?

Reply:
1. I would start with:
"Checked represents true, un-checked represent false."
And then go about using them in groups.
2. Examples of positive commands would be useful – it's hard to imagine how they could sound. Show examples of do's and don'ts.